### ISSUE TITLE
Regression: We can't rename actions in InputMap

### ISSUE THREAD

**Issue by henriiquecampos** [godotengine/godot#33206]
**Godot version:**
3.2.alpha.official

**OS/device including version:**
Solus 4.0 Gnome 3.34.1

**Issue description:**
In 3.1 when we clicked the action name in the Project->Project Settings->Input Map we were able to edit the name of the action, as it seems it is not possible to do this anymore

Ps: As it seems this regression seems to happen very often in Godot (#517, #27113...)

**Steps to reproduce:**
1. Go to Project->Project Settings->Input Map
2. Try to click on an action
3. It doesn't allow you to edit its name

**Minimal reproduction project:**
Any default project

**Comment by jabcross** [comment]
Working on Master. Click the name of the action and press Space.

![image](https://user-images.githubusercontent.com/6402237/67994651-0dfa2e80-fc25-11e9-8032-cf3af0d7c360.png)

**Comment by jabcross** [comment]
But yes, it's weird that the text input opens on the corner and is so tiny. It should replace the label while it's editing.

**Comment by jabcross** [comment]
Pressing Enter also works, which is the standard for MacOS. F2 doesn't.

**Comment by akien-mga** [comment]
IINM double click used to work for that.

**Comment by henriiquecampos** [comment]
On 3.1.1 you just needed a single click

**Comment by Arunkumar1998BOM** [comment]
I have checked this issue in v3.2.alpha.custom_build.9fcd38fc7 (which is basically built from master). I see that we are unable to change the name for default actions (ui_*). But for custom actions I am able to rename the actions.

Tested on :  Ubuntu 18.04.3 LTS 
Suggestion: I guess a double-click to change the name should be better as this operation would feel much more similar (as with other applications), and correspondingly more use-friendly.
I am Open to suggestions
